core/input/input_event.cpp

```diff
@@ -780,7 +780,7 @@ String InputEventMouseMotion::to_string() {
 			break;
 	}
 
-	return "InputEventMouseMotion : button_mask=" + button_mask_string + ", position=(" + String(get_position()) + "), relative=(" + String(get_relative()) + "), speed=(" + String(get_speed()) + "), pressure=(" + rtos(get_pressure()) + "), tilt=(" + String(get_tilt()) + ")";
+	return "InputEventMouseMotion: button_mask=" + button_mask_string + ", position=(" + String(get_position()) + "), relative=(" + String(get_relative()) + "), speed=(" + String(get_speed()) + "), pressure=" + rtos(get_pressure()) + ", tilt=(" + String(get_tilt()) + ")";
```

Comment:
I'd suggest using `vformat()` for all InputEvents to have something more readable and consistent.

Reply:
Done!